### PR TITLE
Nutanix scsi/iscsi tuning

### PR DESCRIPTION
**Pull request by tuxtof**
What this PR does / why we need it:

For Nutanix provider:
- Fix issue with IQN name
- configure SCSI timetout settings for Nutanix Volumes
- set ISCSI tuning settings for Nutanix Volumes

### CODE REVIEW

**Comment by tuxtof** [comment]
Hello @kkeshavamurthy @mboersma 
Is someone can check this PR
thanks

**Review by mboersma — COMMENTED** [review]
/lgtm

File: images/capi/ansible/roles/providers/tasks/nutanix.yml [hunk 1]
```diff
@@ -60,10 +60,35 @@
...
+      node.session.queue_depth: 1024
+      node.session.timeo.replacement_timeout: 240
+      node.conn[0].timeo.noop_out_interval: 5
+      discovery.sendtargets.iscsi.MaxRecvDataSegmentLength: 1048576
+
 - name: Enable iSCSI initiator daemon on Ubuntu or RedHat
   systemd:
     name: iscsid
-    state: started
+    state: stopped
```
[the remark this answers is not in the document]
Comment (tuxtof) [reply images/capi/ansible/roles/providers/tasks/nutanix.yml:91]: that's an excellent question :-D

goal here is to make a template

if we start the service during template build , he create an IQN name who become the same for every node as it is not cleaned in the global sysprep step

in all case there is no reason to start it inside the template

**Pull request merged**